tools/lsp/language/completion.rs

```diff
@@ -664,7 +672,11 @@ fn add_components_to_import(
             );
             result.push(CompletionItem {
                 label: format!("{} (import from \"{}\")", exported_name.name, file),
-                insert_text: Some(format!("{} {{$1}}", exported_name.name)),
+                insert_text: if has_brace {
+                    Some(format!("{}$1", exported_name.name))
```

Comment:
Actually, no need for $1 in that case. Just use a real thing
```suggestion
                    Some(exported_name.name.clone())
```

Reply:
Ah, makes sense. Must be the default behavior.